src/App.js

```diff
@@ -92,6 +92,7 @@ export class App extends React.Component<AppProps, AppState> {
     characterStateSerializer: CharacterStateSerializer;
     allowedActionsSerializer: AllowedActionsSerializer;
     speedLookUp: Array<number>;
+    pushStateTimer: ?TimeoutID;
```

Comment:
I'm thinking renaming this to `pushStateTimeoutID` would make the intention clearer.